modules/client-common/src/main/java/org/apache/ignite/client/proto/query/event/JdbcQueryExecuteRequest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.proto.query.event;
+
+import java.sql.Statement;
+import org.apache.ignite.client.proto.ClientMessagePacker;
+import org.apache.ignite.client.proto.ClientMessageUnpacker;
+import org.apache.ignite.client.proto.query.JdbcStatementType;
+import org.apache.ignite.internal.tostring.S;
+
+/**
+ * JDBC query execute request.
+ */
+public class JdbcQueryExecuteRequest implements JdbcClientMessage {
```

Comment:
Not all fields are serialized.
Is it ok?

Reply:
I removed the non-serializable fields. They are not used now. I thought about keeping them for the future, but realized that's a bad option.